addons/mail/static/src/components/discuss_mobile_mailbox_selection/discuss_mobile_mailbox_selection.xml

```diff
@@ -2,11 +2,11 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.DiscussMobileMailboxSelection" owl="1">
-        <div class="o_DiscussMobileMailboxSelection" t-attf-class="{{ className }}" t-ref="root">
+        <div class="o_DiscussMobileMailboxSelection d-flex" t-attf-class="{{ className }}" t-ref="root">
             <t t-foreach="discussView.discuss.orderedMailboxes" t-as="mailbox" t-key="mailbox.localId">
-                <button class="o_DiscussMobileMailboxSelection_button btn btn-secondary"
+                <button class="o_DiscussMobileMailboxSelection_button btn btn-secondary flex-grow-1"
                     t-att-class="{
-                        'active o-active': discussView.discuss.thread === mailbox,
+                        'active o-active shadow-none': discussView.discuss.thread === mailbox,
```

Comment:
I don't see any difference with or without `shadow-none`, am I missing something?

Reply:
In mobile mode, when any button in the `o_DiscussMobileMailboxSelection` is selected (`:focus`), there is a `box-shadow` assignation added by default in the Bootstrap `.btn` classes, which I think is wrong in this context, giving a weird "outline" effect.

Reply:
Ok, not related to this PR, it's a [FIX].
Good catch, but you should create another commit specifically for it.
I don't know if the guys will agree on merging it in the same PR.. but since it's just a "one line detail", we can make a try.

You can remove the draft state. @odoo/rd-discuss , ready for tech. review 👍

Reply:
> I'm okay with this fix in same PR, as long as it's in its own commit. Just so that the commit doesn't hide this change as an intentional fix. Thanks!

I'm adapting this fix to a commit on its own. Thanks! 👍